mmhuman3d/core/visualization/renderer/torch3d_renderer/base_renderer.py

```diff
@@ -217,30 +266,78 @@ def rgb2bgr(rgbs):
         return bgrs
 
     @staticmethod
-    def image_tensor2numpy(image):
-        return (image.detach().cpu().numpy() * 255).astype(np.uint8)
+    def _normalize(value,
+                   origin_value_range=None,
+                   out_value_range=(0, 1),
+                   dtype=None,
+                   clip=False) -> Union[torch.Tensor, np.ndarray]:
+        """Normalize the tensor or array and convert dtype."""
+        if origin_value_range is not None:
```

Comment:
What if origin_value_range is not None and origin_value_range[0] < value.min()? Will anything go wrong because of this? If so, we could set anything below origin_value_range[0] to origin_value_range[0], also the upper bound.

Reply:
> What if origin_value_range is not None and origin_value_range[0] < value.min()? Will anything go wrong because of this? If so, we could set anything below origin_value_range[0] to origin_value_range[0], also the upper bound.

Origin value range means expected origin value range and out_value_range means expected out value range. E.g., a tensor image is expected to be in [0, 1] and want to normalize it to [0, 255] and write it. It is common that this `tensor.min() >0` if it is bright.

Reply:
Sry, didn't see `if clip:` in L283 before.